.travis.yml

```diff
@@ -2,8 +2,8 @@ language: python
 python:
   - "2.6"
```

Comment:
Are Python 2.6 and Python 3.2 support really needed?
Should we add "Nightly" to get the Python 3.6 builds now that it is in beta?